sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -657,8 +664,9 @@ def extract_endpoints(stage):
           else:
             raise NotImplementedError
           if data_operation_spec:
-            transform.spec.payload = data_operation_spec
-            transform.spec.any_param.CopyFrom(data_operation_spec)
+            beam_fn_api_pb2.RemoteGrpcPort
```

Comment:
What's the purpose of this line?

Reply:
Must be a copy-paste error, thanks for catching.